php/class-wp-seo-settings.php

```diff
@@ -202,7 +202,7 @@ public function get_archived_post_types() {
 	 * @return array With slugs of any enabled taxonomies.
 	 */
 	public function get_enabled_taxonomies() {
-		return $this->get_option( 'taxonomies' );
+		return $this->get_option( 'taxonomies', array() );
```

Comment:
This is calling the class `get_option()`, not core; the class doesn't currently support a default.